common/religion/religions/light_group.txt

```diff
@@ -301,5 +301,52 @@
 			doctrine = tenet_monasticism
 			doctrine = tenet_order_syncretism
 		}
+		#Light/Arcane religion
+		dalaranian_religion = { #Blend between base light religion and arcane religion
+			#Main Group
+			color = rgb { 255 190 230 } #Color is based on holy light with a pinkish tint
+			religious_head = k_dalaran #King of Dalaran should be head of faith
```

Comment:
Dalaran first of all is a secular landed kingdom. It can't lead the faith. So you have to create a new title (tell me if you don't know how to do it).
Also, we had a discussion in the dev chat. I personally would like to have The Kirin Tor as a head, but it may be The Six as other dev suggested.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/113424018-83e96300-93e0-11eb-911d-dac2830b9da8.png)

</details>